tests/page-objects/dataTable.ts

```diff
@@ -0,0 +1,77 @@
+import { Page, Locator, expect } from "@playwright/test"
+import { Dialog } from "puppeteer";
+
+export class dataTableElements{
+    protected page: Page;
+    readonly dataTable
+    readonly table: Map<number,{editBtn:Locator,deleteBtn:Locator,nameEdit:Locator,emailEdit:Locator,saveBtn:Locator, name:Locator, email:Locator}>
+   
+    constructor(page: Page) {
+        this.page = page;
+        this.dataTable = this.page.getByTestId('dataTable');
+    
+        this.table = new Map([
```

Comment:
Also you'll need to explain this - I'm not sure that this is the best thing to do for a table. Maybe it is.